src/datasource.js

```diff
@@ -933,7 +933,9 @@ ngeo.DataSource.OGCType = {
  * @enum {string}
  */
 ngeo.DataSource.WFSFeatureNS = {
-  MAPSERVER: 'http://mapserver.gis.umn.edu/mapserver'
+  GEOSERVER: 'http://www.opengis.net/gml/3.2',
```

Comment:
I can't test that. Is this wfsFeatureNS link correct ? (the geoserver one, qgis and mapserver work)